docs/topics/tour/kotlin-tour-functions.md

```diff
@@ -161,11 +161,52 @@ fun main() {
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-tour-simple-function-after"}
 
-> Omitting the return type is only possible when your function has no body (`{}`). Unless your function's return type
-> is `Unit`.
+However, if you want your code to be quickly understood by other developers, it's a good idea to explicitly define the 
+return type even when using the assignment operator `=`.
+
+> If you use `{}` curly braces to declare your function body, you must declare the return type unless it is `Unit`.
```

Comment:
```suggestion
> If you use `{}` curly braces to declare your function body, you must declare the return type unless it's a `Unit`.
```

Reply:
What about "it is `Unit` type"?

Reply:
OK, but shouldn't it be "it is the `Unit` type" in this case?